src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporterHelperExtensions.cs

```diff
@@ -28,6 +25,7 @@ public static class OtlpLogExporterHelperExtensions
     /// <remarks><inheritdoc cref="AddOtlpExporter(OpenTelemetryLoggerOptions, Action{OtlpExporterOptions})" path="/remarks"/></remarks>
     /// <param name="loggerOptions"><see cref="OpenTelemetryLoggerOptions"/> options to use.</param>
     /// <returns>The instance of <see cref="OpenTelemetryLoggerOptions"/> to chain the calls.</returns>
+    [Obsolete("Call LoggerProviderBuilder.AddOtlpExporter instead this method will be removed in a future version.")]
```

Comment:
👍 to obsoleting these methods now. Obsoleting is the signal used to push users to new, preferred APIs. Without the signal, it remains unclear which API to use. Obsoleting is better than alternatives like relying on people to read release notes or intellisense documentation - they often don't.